src/test/unit/actions/searchDescriptionActions/searchDescription.test.ts

```diff
@@ -0,0 +1,11 @@
+import { SearchDescriptionActions } from '../../../../main/resources/actions/searchDescriptionActions';
+
+const searchDescriptionActions = new SearchDescriptionActions();
+
+describe('getStatusDescriptionList()', () => {
+  const results = searchDescriptionActions.getStatusDescriptionList();
+
+  it('should return list of 49 courts events status', () => {
+    expect(results.length).toBe(49);
+  });
```

Comment:
youre not testing the `else` block of your code

Reply:
still need more tests here

Reply:
@junaidiqbalmoj  i still cant see where youre testing the else block of your code

Reply:
I have added more tests to this file. There was only one test before, now I have added 3 more tests.